test/FunctionalTestUtils20/HttpListenerObservableBase.cs

```diff
@@ -58,7 +58,7 @@ public void Dispose()
         {
             if (listener != null && listener.IsListening)
             {
-                listener.Stop();
+                //listener.Stop();
```

Comment:
is it a leftover of local testing? Do not keep listeners opened

Reply:
There is close() method called right after this which will shutdown the listener anyway so using Stop is not required.
And using both throws exception in linux about 'Address already in use'.